Dockerfile

```diff
@@ -1,6 +1,9 @@
 # https://hub.docker.com/_/microsoft-dotnet
 FROM mcr.microsoft.com/dotnet/sdk:5.0.102-ca-patch-buster-slim AS build-env
 
```

Comment:
Trying to re-wrap my head around what this does, but is this build configuration needed?